index.html

```diff
@@ -1,119 +1,125 @@
+
+                <div class="dino__front-leg"></div>
+
+            </div>
+
+            <div class="floor"></div>
+            <!-- ====== End of dinosaur========= -->
+
+            <h1>Event Countdown</h1>
+            <h2>100 <span>Days Of Code</span></h2>
+            <div class="countdown">00 : 00 : 00 : 00</div>
+
+            <a href="https://forms.office.com/r/L2U0x4Qzdp" target="_blank" class="RegisterButton">REGISTER</a>
+            <h4 class="faq"><a href="#faq">FAQ</a></h4>
         </div>
-      
-        <div class="dino__back-leg"></div>
-      
-        <div class="dino__tail"></div>
-      
-        <div class="dino__bottom"></div>
-        <div class="dino__bottom__shadow"></div>
-      
-        <div class="dino__front-leg"></div>
-      
-      </div>
-      
-      <div class="floor"></div>
-<!-- ====== End of dinosaur========= -->
-
-      <h1>Event Countdown</h1>
-      <h2>100 <span>Days Of Code</span></h2>
-      <div class="countdown">00 : 00 : 00 : 00</div>
-
-      <a href="https://forms.office.com/r/L2U0x4Qzdp" target="_blank" class="RegisterButton">REGISTER</a>
-    <h4 class="faq"><a href="#faq"  >FAQ</a></h4>
-    </div>
-  </header>
+    </header>
+
 
     <!--Frequently Asked Questions-->
-    <div class="accordion-body" id="faq">
-    <div class="accordion">
-      <div class="accordion-text">
-        <div class="title">Frequently Asked Questions</div>
-      <ul class="faq-text">
-        <li>
-          <div class="question-arrow">
-            <span class="question">What is #100DaysOfCode challenge?</span>
-            <i class="bx bxs-chevron-down arrow"></i>
-          </div>
-          <p>The 100 Days Of Code challenge encouraging beginners to start to learn code, encourage them to post a tweet about what they did and use the hashtag of #100DaysOfCode and submit the final project after the event into <a href="https://github.com/SurPathHub/showcase-your-project">showcase-your-project</a> repository.</p>
-          <span class="line"></span>
-          <li>
-            <div class="question-arrow">
-              <span class="question">Is it required to post tweet everyday?</span>
-              <i class="bx bxs-chevron-down arrow"></i>
+    <div class="accordion-body">
+        <div class="accordion">
+            <div class="accordion-text">
+                <div class="title">Frequently Asked Questions</div>
+                <ul class="faq-text">
+                    <li>
+                        <div class="question-arrow">
+                            <span class="question">What is #100DaysOfCode challenge?</span>
+                            <i class="bx bxs-chevron-down arrow"></i>
+                        </div>
+                        <p>The 100 Days Of Code challenge encouraging beginners to start to learn how to code, encourage them to post a tweet about what they did and use the hashtag of #100DaysOfCode and submit the final project after the event into <a class="links"
```

Comment:
`Change it to: `
The 100 Days Of Code challenge encouraging beginners to start to learn code